test/e2e/config/docker.yaml

```diff
@@ -5,17 +5,17 @@
 # - control-plane kubeadm
 # - docker
 
-# For creating local dev images run ./scripts/ci-e2e.sh
+# For creating local dev images run make docker-build-e2e from the main CAPI repository
```

Comment:
from @sedefsavas https://github.com/kubernetes-sigs/cluster-api/pull/3639#discussion_r489147384

> I think we can keep run ./scripts/ci-e2e.sh here, because running make docker-build-e2e will not create images with the tag dev this file is using.

Uhmm, the default in the make file is TAG=dev
https://github.com/kubernetes-sigs/cluster-api/blob/19e40cd536d5ea0296599e2678e7cc70e385b452/Makefile#L88
https://github.com/kubernetes-sigs/cluster-api/blob/19e40cd536d5ea0296599e2678e7cc70e385b452/test/infrastructure/docker/Makefile#L50

Reply:
LOL, I've done the same in #3593 after much yak shaving.